.github/scripts/trymerge.py

```diff
@@ -1390,6 +1427,11 @@ def merge(pr_num: int, repo: GitRepo,
     # here to stop the merge process right away
     find_matching_merge_rule(pr, repo, skip_mandatory_checks=True)
 
+    if not has_required_labels(pr):
+        raise RuntimeError(LABEL_ERR_MSG)
```

Comment:
Then this bit would change to:

```suggestion
        raise RuntimeError(MERGE_LABEL_ERR_MSG)
```